pom.xml

```
@@ -966,5 +966,6 @@
     <maven-license-plugin.version>1.8</maven-license-plugin.version>
     <incrementals-plugin.version>1.2</incrementals-plugin.version>
     <incrementals-enforce-minimum.version>1.0-beta-4</incrementals-enforce-minimum.version>
+    <argLine />
```

BTW the space is unnecessary as that is only used by HTML, not XML
```suggestion
    <argLine/>
```
but IIRC the crappy XML round-trip library used by `release:prepare` forcibly reinserts the space anyway, hence its prevalence in `<relativePath />`.